storage/innobase/srv/srv0start.cc

```diff
@@ -2581,18 +2581,30 @@ dberr_t srv_start(bool create_new_db) {
 
     srv_dict_metadata = recv_recovery_from_checkpoint_finish(*log_sys, false);
 
+    // Disabled for the clone cross-engine synchronization: the metadata updates
+    // are redo-logged and the clone will take the ones up to the clone LSN.
+    // Any metadata past that point is incosistent. Thus keep the metadata we
```

Comment:
Nit: inconsistent

Reply:
Fixed (and tweaked a comment a bit in general)